cypress/support/pages/user.pageObject.js

```
@@ -0,0 +1,22 @@
+import PageObject from '../PageObject';
+
+class UserPageObject extends PageObject {
+  constructor() {
+    super();
+    this.followBtn = () => cy.get('[data-qa="follow-btn"]');
+  }
+
+  visitProfile(username) {
+    super.visit(`/profile/${username}`);

```

visitProfile correctly builds the profile URL, but the method should also assert navigation succeeded (helps tests fail fast and makes flows deterministic). Consider adding an assertion such as `cy.url().should('include', `/profile/${username}`)` or verifying a profile heading is visible after navigation.